<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunnAllDomain</class>
 <widget class="QDialog" name="RunnAllDomain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RunnAllDomain</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>5</y>
     <width>571</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QPushButton" name="pushButtonConstrain">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>36</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditConstrain">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Input Constraining layer for Domain Decomposition. Generally, an output shapefile produced after Vector Merge.</string>
    </property>
    <property name="text">
     <string>Constrain Layer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>173</y>
     <width>571</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QPushButton" name="pushButtonMesh">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>37</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMesh">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Specify a Shape file where TINs generated by TRIANGLE can be save.</string>
    </property>
    <property name="text">
     <string>Mesh (TIN) File</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>90</y>
     <width>571</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QCheckBox" name="checkBoxAngle">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>40</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Quality 'Angle' constraint for TIN generation</string>
    </property>
    <property name="text">
     <string>Angle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAngle">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxArea">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>36</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Maximum Area Quality constraint for TIN generation.</string>
    </property>
    <property name="text">
     <string>Area (m^2)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditArea">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOthers">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Other TRINAGLE options</string>
    </property>
    <property name="text">
     <string>Others</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOthers">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>268</y>
     <width>571</width>
     <height>53</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButtonRun">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>12</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>12</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHelp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
  </widget>
  <widget class="Q3TextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>337</y>
     <width>571</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxAngle</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditAngle</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxArea</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditArea</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxOthers</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditOthers</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
